src/tests/multihost/alltests/test_services.py

```diff
@@ -200,3 +202,98 @@ def test_0006_bz1909755(self, multihost, backupsssdconf):
                                    'to signal service .* No '
                                    'such file or directory" '
                                    '/var/log/sssd')
+
+    @pytest.mark.tier1_2
+    def test_0007_bz971435(self, multihost, backupsssdconf):
+        """
+        :title: Enhance sssd init script so that it would source a configuration
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=971435
+        :id: ba764db9-90e0-4e6a-bd3c-b7bd6221f340
+        :steps:
+          1. Add 'PIZZA=YUMMY' string in /etc/sysconfig/sssd
+          2. Restart the sssd
+          3. Check pid of sssd and grep the string under /proc directory
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+        """
+        # port bash to pytest
+        tools = sssdTools(multihost.client[0])
+        sssd_params = {'debug_level': '9'}
+        tools.sssd_conf('sssd', sssd_params)
+        exceute_cmd(multihost, f'echo "PIZZA=YUMMY" > /etc/sysconfig/sssd')
```

Comment:
`f` is not needed here, it is causing flake8 error

Reply:
Added multihost.client[0].run_command.